ui/raidboss/raidboss_config.ts

```diff
@@ -1416,7 +1507,8 @@ const templateOptions = {
         en: 'e8s: enable cactbot Uptime Knockback strat',
         de: 'e8s: aktiviere cactbot Uptime Knockback Strategie',
         fr: 'e8s : activer cactbot pour Uptime Knockback strat',
-        ja: 'エデン零式共鳴編４層：cactbot「ヘヴンリーストライク (ノックバック)」ギミック',
+        ja:
+            'エデン零式共鳴編４層：cactbot「ヘヴンリーストライク (ノックバック)」ギミック',
```

Comment:
```suggestion
        ja: 'エデン零式共鳴４層：「ヘヴンリーストライク (ノックバック)」ギミック',
```

to avoid multiple lines